pkg/helm/fake.go

```diff
@@ -97,7 +97,9 @@ func (c *FakeClient) InstallReleaseFromChart(chart *chart.Chart, ns string, opts
 	}
 
 	release := ReleaseMock(&MockReleaseOptions{Name: releaseName, Namespace: ns, Description: releaseDescription})
-	c.Rels = append(c.Rels, release)
+	if !c.Opts.dryRun {
+		c.Rels = append(c.Rels, release)
```

Comment:
I think we still want to display the "deployed" release in the response, no?

Reply:
Hm, not sure I understand what you mean; with this change, a dry-run release is returned in the InstallReleaseResponse but isn't remembered by the "server", same as real code.

Reply:
oh gotcha. That's what I get for reading code prior to coffee. ☕️

Reply:
oh gotcha. That's what I get for reading code prior to coffee consumption. ☕️